solidity/random-beacon/README.adoc

```diff
@@ -55,39 +55,38 @@ Instead of a v1 ticket-based approach for group selection, we use a sortition
 pool. Group creation start transaction is embedded into relay request
 transaction and locks a sortition pool. From this moment, no operator can enter
 or leave the pool. Once a new relay entry appears on the chain, all off-chain
-clients perform group selection optimistically calling
-`RandomBeacon.selectGroup(seed)` view function for free. Seed is available in
-`DkgStarted` event emitted when the group creation starts. After determining
-group members, clients should perform off-chain distributed key generation (DKG).
- <<dkg-submit-eligibility,Eligible group member>> submits the result to the chain
- calling `RandomBeacon.submitDkgResult(DKG.Result calldata dkgResult)` function.
- Once the result is submitted, a challenge period starts.
+clients perform group selection by calling `RandomBeacon.selectGroup()` view
+function for free. After determining group members, clients should perform
+off-chain distributed key generation (DKG). One of the group members submits
+the result to the chain calling
+`RandomBeacon.submitDkgResult(DKG.Result calldata dkgResult)` function.
+Once the result is submitted, a challenge period starts.
 
 During the challenge period, anyone can notify that the submitted DKG result is
-malicious calling `RandomBeacon.challengeDkgResult(DKG.Result calldata dkgResult)`
+malicious by calling `RandomBeacon.challengeDkgResult(DKG.Result calldata dkgResult)`
 function. A malicious DKG result may contain corrupted data, group members not
 selected by the pool, or incorrect supporting signatures. If such malicious
-result is submitted and successfully challenged, the result submitter gets slashed and the
-malicious result is immediately discarded.
-
-Once the challenge period passes, and no challenges are reported,
-the DKG result submitter should unlock the sortition pool and mark the DKG result as
-accepted calling
-`RandomBeacon.approveDkgResult(DKG.Result calldata dkgResult)` to receive a
-reward. In case the submitter does not call the approve function within a
-specific number of blocks, anyone can do that and receive the
-submitter's reward as described in <<fees-and-rewards,Fees and Rewards>> section.
-
-There is a timeout before which a DKG result should be submitted. The timeout
-equals the group size multiplied by the number of blocks for a member to become
-eligible to submit a DKG result. The timer starts at the moment when the first
-member becomes eligible.
-
+result is submitted and successfully challenged, the result submitter gets
+slashed and the malicious result is immediately discarded.
```

Comment:
Maybe it would be worth to add that
- submitter of the successful challenge gets some percentage of the slashed amount
- after the result gets discarded the members of the group are given another chance to publish the result and DKG timeout timer is reset

Reply:
07ad25c3b3bf9c7f2a23853a0b071397ee62a873